src/main/java/uk/gov/mca/beacons/api/configuration/SecurityConfiguration.java

```diff
@@ -81,7 +81,12 @@ protected void configure(AuthenticationManagerBuilder auth)
     @Override
     protected void configure(HttpSecurity http) throws Exception {
       super.configure(http);
-      http.cors().and().authorizeRequests().antMatchers("/**").authenticated();
+      http
+        .cors()
+        .and()
+        .authorizeRequests()
+        .antMatchers("/spring-api/**")
```

Comment:
Move just the path `/spring-api` behind AzureAd security.  This frees up paths on `/` for serving the Webapp.